### PR TITLE
New file added

### PR DESCRIPTION
**Pull request by jayeshsony**

### CODE REVIEW

File: practical.sql [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+)
+
+CREATE TABLE `employee_salary` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `fk_employee_id` int NOT NULL,
+  `salary` double DEFAULT NULL,
+  `salary_date` date NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `employee_salary_fk` (`fk_employee_id`),
+  CONSTRAINT `employee_salary_fk` FOREIGN KEY (`fk_employee_id`) REFERENCES `employee` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
```
Comment (KrushaliTalaviya) [inline practical.sql:24]: Please Remove  ON DELETE CASCADE ON UPDATE CASCADE from all

File: practical.sql [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+	   ('Naitik', 'Bogdi', 23, 8278545389, 'Paladi, Ahemdabad'),
+	   ('Harsh', 'Chaudhary', 25, 9876433254, 'Ayodhyanagar Society, Palanpur'),
+	   ('Anand', 'Pate', 26, 8765349201, 'Chandralock Society, Palanpur');
+
+INSERT INTO employee_salary (fk_employee_id, salary, salary_date)
+VALUES (1, 15000, '2024-03-15'), (2, 10000, '2024-03-10'), (3, 18000, '2024-04-24'), (4, 20000, '2024-04-12'), (1, 15000, '2024-04-15'), (2, 10000, '2024-04-10');
+
+INSERT INTO employee_hobby (fk_employee_id, fk_hobby_id)
+VALUES (1, 2), (2, 3), (3, 1), (4, 4);
+
```
Comment (KrushaliTalaviya) [inline practical.sql:52]: Where is the Update Query?

File: practical.sql [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+SELECT *
+FROM employee;
+
+SELECT *
+FROM employee_salary;
+
+SELECT *
+FROM employee_hobby;
+
+SELECT concat(first_name, ' ', last_name) AS 'Employee Name'
```
Comment (KrushaliTalaviya) [inline practical.sql:105]: Format This query
As well as also change the name of the alias, We can use name like employee_name

**Review by KrushaliTalaviya — CHANGES_REQUESTED** [review]

**Comment by jayeshsony** [comment]
Added  comments and formatted queries.

**Comment by jayeshsony** [comment]
Added details and formatted the queries.

**Comment by jayeshsony** [comment]
Added comments and formatted the queries.

**Review by KrushaliTalaviya — APPROVED** [review]

**Pull request merged**